src/stats/BasicRFunctions.java

```diff
@@ -51,7 +66,14 @@ public static double StudentKnowledgeEstAvg(KnowledgeEstimateMatrix loMatrix, St
         return actual;
     }
 
-    //Must be called at the start of every function that uses RCaller methods
+    /**
+     * Must be called at the start of every function that uses RCaller methods in
+     * order to properly run RCaller methods. The if statement checks whether the machine
+     * running the function is Windows or Mac and sets up RCaller accordingly.
+     * @pre called at some point before RCode is set and returned
+     * @return an RCaller that works on whichever machine is running
+     */
+
     public static RCaller RCallerVariable(){
```

Comment:
LEts talk ablout this style of this function from a functional programming perspective.